### PR TITLE
fix (argocd-notifications): Slack notification service is disabled in values.

### PR DESCRIPTION
**Pull request by 1Psy**
Checklist:

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by oliverbaehler** [comment]
@1Psy Hi, Thanks for your contribution. Could you give some more context on why this should not be commented in the default values?

**Comment by 1Psy** [comment]
@oliverbaehler Hi. I would like to disable it because it records to CM even if I use another notifier.
slack

File: charts/argocd-notifications/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ appVersion: 1.0.2
 description: A Helm chart for ArgoCD notifications, an add-on to ArgoCD.
 name: argocd-notifications
 type: application
-version: 1.0.15
+version: 1.0.16
```
Comment (oliverbaehler) [inline charts/argocd-notifications/Chart.yaml:6]:
```suggestion
version: 1.1.3
```

**Review by oliverbaehler — CHANGES_REQUESTED** [review]
Verison Bump

**Comment by 1Psy** [comment]
I want to refactor the chat and add this in the future.

**Pull request closed without merging**